src/app/api/routing.py

```diff
@@ -20,11 +23,11 @@ async def get_entry(table: Table, entry_id: int = Path(..., gt=0)):
     return entry
 
 
-async def fetch_entries(table: Table, query_filter: Optional[Tuple[str, Any]] = None):
+async def fetch_entries(table: Table, query_filter: Optional[List[Tuple[str, Any]]] = None):
```

Comment:
Good thinking, I hesitated to suggest you to do this for your location update suggestion :)

Reply:
That's cleaner and more versatile this way :) 
But we will need something else if we end up using "or" clauses.